src/toil_scripts/gatk_processing/gatk_preprocessing.py

```diff
@@ -521,91 +330,111 @@ def indel_realignment(job, shared_ids, input_args):
 
     docker_call(tool='quay.io/ucsc_cgl/gatk:3.5--dba6dae49156168a909c43330350c6161dc7ecc2',
                 work_dir=work_dir, parameters=parameters,
-                inputs=['ref.fa', 'sample.mkdups.bam', 'phase.vcf', 'mills.vcf',
-                        'sample.intervals', 'ref.fa.fai', 'ref.dict',
-                        'sample.mkdups.bam.bai'],
-                outputs={'sample.indel.bam': None, 'sample.indel.bai': None},
-                env={'JAVA_OPTS':'-Xmx10g'})
+                inputs=inputs, outputs=outputs,
+                env={'JAVA_OPTS':'-Xmx%sg' % memory},
+                mock=config['mock'])
 
     # Write to fileStore
-    shared_ids['sample.indel.bam'] = job.fileStore.writeGlobalFile(outpath)
-    _move_bai(outpath)
-    shared_ids['sample.indel.bam.bai'] = job.fileStore.writeGlobalFile(outpath + ".bai")
-    job.addChildJobFn(base_recalibration, shared_ids, input_args, cores = input_args.cpu_count)
+    bam_id = job.fileStore.writeGlobalFile(outpath_bam)
+    bai_id = job.fileStore.writeGlobalFile(outpath_bai)
+    return bam_id, bai_id
 
 
-def base_recalibration(job, shared_ids, input_args):
+def gatk_base_recalibrator(job, bamFileStoreID, baiFileStoreID, config, xmx=8):
```

Comment:
This exists in `toil_scripts.tools.preprocessing` already; remove/consolidate differences.